### ISSUE TITLE
Should the self property of the self link not include query params?

### ISSUE THREAD

**Issue by rduivenvoorde** [CrunchyData/pg_featureserv#144]
Hi, experimenting with a rather large db, which can practically (because of size) only be queried by using a spatial and temporal filter:

```
http://localhost:9000/collections/test.featureserv_metingen/items.json?limit=10000&bbox=4.44333975882750654,50.94963634583672274,5.44794455106176745,51.39568604304270849&filter=starttime%20%3E%20%272023-10-11T07%3A09%3A00.000%2B00%3A00%27%20and%20starttime%20%3C%20%272023-10-11t07%3A11%3A00.000%2B00%3A00%27
```

while bbox and filter work (in this case returning 1 query), the 'self' link in the result shows the full url: http://localhost:9000/collections/test.featureserv_metingen/items.json

I'm wondering if this should not include the other query params (same like when you are paging around in a dataset)?

**Comment by dr-jts** [comment]
Yes, the `self` link should include all the query parameters.  This is a bug.
